includes/admin.form.inc

```diff
@@ -94,6 +94,14 @@ function islandora_compound_object_admin_form($form, &$form_state) {
     '#default_value' => variable_get('islandora_compound_object_show_compound_parents_in_breadcrumbs', FALSE),
   );
 
+  $form['islandora_compound_object_redirect_to_first'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Redirect to first child when a compound object is requested.'),
+    '#description' => t('Compound objects themselves redirect to there first child when this setting is enabled. This solves the problem that blocks on compound views get the wrong context.'),
```

Comment:
I would suggest
` t('Compound objects redirect to their first child when this setting is enabled') `
and I would remove the reason for this.
OR

 `t ('Users will be redirected to a Compound's Object first child when enabling this setting.'),`

OR

`t ('Users will be redirected to the first child of a Compound Object when enabling this setting.'),`

OR

@whikloj @jonathangreen you are native English speakers. Advice/corrections?

Reply:
I like option C
`t ('Users will be redirected to the first child of a Compound Object when enabling this setting.'),`

Reply:
Not to complicate things here even further, but the standard practice on these checkboxes is for the label to be a statement indicating what happens when the checkbox is checked. For instance, in Scholar the checkbox labels are "Enable RoMEO attempts", "Render Google Scholar Search link", "Let users choose display CSL", etc. 

I would recommend having this type of statement for the checkbox, for example "Redirect to first child of Compound Objects" or similar.

Reply:
@DiegoPino @whikloj Should I change the title of the checkbox to `t ('Users will be redirected to the first child of a Compound Object when enabling this setting.'),` ? Leave the description or remove it? Any other things I should change?
I have a feeling this is an ongoing discussion where someone needs to make a decision.
I don't mind changing the PR, but like to reduce the number of times that I need to do that. So if I know exactly what to do, I will add it in another commit. Please let me know.

Reply:
@lucasvanschaik Change the `#description` of the checkbox to `t('Users will be redirected to the first child of a Compound Object when enabling this setting.')`

Reply:
and sorry for the delay in responding :man_facepalming: